core/tests/protractor.conf.js

```diff
@@ -338,6 +339,27 @@ exports.config = {
       }));
     }
 
+      // Adding a video reporter. For more information see
+      // https://www.npmjs.com/package/protractor-video-reporter
+    var _ADD_VIDEO_REPORTER = true;
+
+    var ffmpegArguments = [
+      '-y',
+      '-r', '30',
+      '-f', 'x11grab',
+      '-s', '1366x768',
+      '-i', process.env.DISPLAY,
+      '-g', '300',
+      '-vcodec', 'qtrle',
+    ]
+
+    if(_ADD_VIDEO_REPORTER) {
+      jasmine.getEnv().addReporter(new VideoReporter({
+        baseDirectory: path.resolve(__dirname, '../protractor-video'),
```

Comment:
Did you try setting `singleVideoPath: 'fullName'`? That might address the problem of suites' videos overwriting each other. Another idea: you could set it to `'uuid'` and print out which UUID goes with which test

Reply:
I've tried both options, but for some reason the package keeps using `protractor-specs.mov`

Reply:
Have you tried logging `currentSpec.fullName` and `finalFileName`?

Reply:
I have done the above already. For now, I have gotten `uuid` working so we will be using that (the `uuid` for each video can be found in the logs).